index.html

```diff
@@ -1,26 +1,113 @@
+    laptop.</p>
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Flike.png?v=1603327411736" alt="">
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Freadon.png?v=1603325721697" alt=""> 
+
+<section> 
+  <ul> For the Donuts 
+    <li>3 tablespoons active dry yeast or 2 1/4 teaspoons instant yeast</li>
+    <li>1 cup whole milk, heated to 110˚F (43°C)</li>
+    <li>2 to 2 1/2 cups bread flour, plus a bit for the work surface</li>
+    <li>2 tablespoons superfine sugar (or just blitz granulated sugar in a blender until finely ground)</li>
+    <li>1/2 teaspoon salt</li>
+    <li>1 teaspoon vanilla extract</li>
+    <li>3 large egg yolks</li>
+    <li>4 tablespoons unsalted butter (2 oz), at room temperature</li>
+    <li>Vegetable oil, for frying</li>
+  </ul> 
+
+  <ul>For the Glaze 
+    <li
+  </ul>
+</section>
+
+  <section>
+
+    <ol> Donuts
+      <li>1. In a medium bowl, dissolve 2 tablespoons yeast in 3/4 cup warm milk. Stir in 3/4 cup of the flour to create
+        a smooth paste. Cover and let rest in a warm spot for 30 minutes.</li>
+      <li>2. Combine the remaining warm milk and yeast in the bowl. Add the flour mixture along with the sugar, salt,
+        vanilla, and egg yolks. Mix until smooth. Turn off the mixer and add 1/2 cup of the remaining flour. Mix on low
+        for about 30 seconds. Add the butter and mix until it becomes incorporated, about 30 seconds. Add more flour,
+        about 1/4 cup at a time. Knead until the dough pulls completely away from the sides of the bowl and is smooth
+        and not too sticky. Cover the bowl with plastic wrap and refrigerate for at least 1 hour and up to 12 hours.
+      </li>
+      <li>3. Line a baking sheet with a lightly floured dish towel. Roll out the dough on a lightly floured surface to
+        1/2 inch thick. With a donut or cookie cutter, cut out 3-inch-diameter rounds with 1-inch-diameter holes. You
+        can re-roll the scraps and cut out additional holes.</li>
+      <li>4. Place the donuts at least 1 inch apart on the baking sheet and cover loosely with plastic wrap. Let rest in
+        a warm spot to proof until they almost double in size, 5 to 20 minutes, peeking every five minutes. To test
+        whether the dough is ready, touch it lightly with a fingertip. If it springs back immediately, it needs more
+        time. If it springs back slowly, it is ready.</li>
+      <li>5. While the donuts are proofing, heat a heavy-bottomed pot with at least 2 inches of oil until a deep-fat
+        thermometer registers 360˚F (182°C). With a metal spatula, carefully place a couple of doughnut holes or
+        doughnuts in the oil, being careful not to crowd the pot. Fry for 1 to 2 minutes per side, until light golden
+        brown. Remove and drain on a wire rack over a paper towel, and let cool slightly before glazing. Repeat with the
+        remaining donuts and holes.</li>
+    </ol>
```

Comment:
You can remove the numbers here, as the `ol` creates them for you:

<img width="318" alt="Screen Shot 2020-10-22 at 9 29 53 AM" src="https://user-images.githubusercontent.com/2653980/96878802-3fc56000-1449-11eb-9e59-d749aac00272.png">